pyincore_viz/analysisviz.py

```diff
@@ -19,6 +19,9 @@ def visualize(dataset, **kwargs):
                 None
 
         """
+        # data types that needs to use pop_results_table visualization
+        pop_result_table_data_types = ['incorehousingunitallocation']
```

Comment:
We may need to think about how we can maybe make this more dynamic, some way to "register" additional types without modifying the code itself.

Reply:
Not for now, but as an improvement we may want to see if we can come up with a more dynamic way to do this.

Reply:
that is exactly what I was thinking but I was not able to find any better solution. Do you have any suggestion?

Reply:
we have very similar situation in incore service for registering hazard data type but it is using the same way. Just put datatypee list in the code